demo/admin/src/common/ContentScopeProvider.tsx

```diff
@@ -59,13 +59,20 @@ const ContentScopeProvider: React.FC<Pick<ContentScopeProviderProps, "children">
     const allowedSiteConfigs = Object.fromEntries(
         Object.entries(sitesConfig.configs).filter(([siteKey, siteConfig]) => allowedUserDomains.includes(siteKey)),
     );
-    const values: ContentScopeValues<ContentScope> = {
-        domain: Object.keys(allowedSiteConfigs).map((key) => ({ value: key })),
-        language: [
-            { label: "English", value: "en" },
-            { label: "German", value: "de" },
-        ],
-    };
+    const values: Array<ContentScopeValues> = [
```

Comment:
I'd find it better if you would adapt `ContentScopeValues` to always be an array. Something like:

```ts
export type ContentScopeValues<S extends ContentScopeInterface = ContentScopeInterface> = Array<{
    [P in keyof S]: Array<{ label?: string; value: NonNull<S[P]> }>;
}>;
```

Also, please don't remove the generic here